single.php

```diff
@@ -38,6 +38,7 @@ function add_body_classes_for_post( $classes ) {
 $page_terms_data             = get_the_terms( $post, 'p4-page-type' );
 $context['author_override']  = $page_meta_data['p4_author_override'][0] ?? '';
 $context['background_image'] = $page_meta_data['p4_background_image_override'][0] ?? '';
+$context['post_image_id']    = $page_meta_data['p4_background_image_override_id'][0] ?? $page_meta_data['_thumbnail_id'][0];
```

Comment:
I'm not sure, was this meant to be p4_background_image_override instead of p4_background_image_override_id?

Reply:
nop, I wanted the id